asgi_correlation_id/extensions/sentry.py

```diff
@@ -22,7 +22,11 @@ def set_transaction_id(correlation_id: str) -> None:
     The transaction ID is displayed in a Sentry event's detail view,
     which makes it easier to correlate logs to specific events.
     """
-    from sentry_sdk import configure_scope
+    import sentry_sdk
 
-    with configure_scope() as scope:
+    if sentry_sdk.VERSION >= '2.12.0':
```

Comment:
Is 2.12.0 the version where the warning was added, or when the `get_isolation_scope` function was added? It seems like it would be best to use the version where `get_isolation_scope` was added, doesn't it?

Reply:
The method `get_isolation_scope` was added in 2.0 but only in the class Scope, it wasn't added in the __init__.py, that means we must use `sentry_sdk.Scope.get_isolation_scope()` to use the method. The warning and the function was added in __init__.py in version 2.12, that's why I did this.

Reply:
You can check it here:
Added in __init__.py https://github.com/getsentry/sentry-python/blob/2.12.0/sentry_sdk/__init__.py
Doesn't added in __init__.py https://github.com/getsentry/sentry-python/blob/2.11.0/sentry_sdk/__init__.py

Added in 2.0 but without the warning, https://github.com/getsentry/sentry-python/blob/2.0.0/sentry_sdk/scope.py#L273
Warning added in 2.12: https://github.com/getsentry/sentry-python/blob/2.12.0/sentry_sdk/api.py#L223